src/lib/data.ts

```diff
@@ -645,6 +649,8 @@ function load_from_local_storage_and_sync() {
                 const key = full_key.substr(store_prefix.length);
                 try {
                     remote_store[key] = JSON.parse(localStorage.getItem(full_key)) as RemoteKV;
+                    store[key] = remote_store[key].value;
```

Comment:
Does this break `LOCAL_OVERWRITES_REMOTE` ? At a glance it seems like it might, but I'm just waking up over here.

Reply:
Hmm - yeah, maybe it should not be done in that case: definitely needs thought

Reply:
( but then - no-one is using LOCAL_OVERWRITES_REMOTE, so....  ;) :o )

Reply:
... yeah, it definitely should not be done if LOCAL_OVERWRITRS_REMOTE.